### DISCUSSION TITLE
How to decode the changes0.json in diff folder under new.pptx-hist?

### DISCUSSION THREAD

**Discussion by XianglongTan** [ONLYOFFICE/DocumentServer#3070] [Q&A]
[
    "66;AgAAADEA//8BAHe2AugW5gAAzwAAAAIAAAAAAAAAAAAAAAAAAAAAAAAA9v///xAAAAA4AC4AMgAuADIALgAyADIA",
    "28;CAAAADEANAA2ADkAAQBABAAAAACZBzEBxy2XAA==",
    "28;CAAAADEANAA2ADkAAgBABAAAAAChdRcAoXUXAA=="
]

This is an example. I guess it records the user operation but I find it hard to decode these strings. Someone have succeeded?

**Comment by zamentur** [comment]
You can decode the second part of each line with `base64 -d` or any base64 decode tools.

However, i'm affraid it's still difficult to recreate a lost file from this.
